src/cli/Command.php

```diff
@@ -127,13 +127,16 @@ public function actionExec($id, $ttr, $attempt)
      * @throws
      * @see actionExec()
      */
-    private function handleMessage($id, $message, $ttr, $attempt)
+    protected function handleMessage($id, $message, $ttr, $attempt)
     {
         // Executes child process
+        $queueCmd = (\Yii::$app->id != $this->module->id)
+            ? $this->module->id.'/'.$this->id
+            : $this->id;        
         $cmd = strtr('{php} {yii} {queue}/exec "{id}" "{ttr}" "{attempt}"', [
             '{php}' => PHP_BINARY,
             '{yii}' => $_SERVER['SCRIPT_FILENAME'],
-            '{queue}' => $this->id,
+            '{queue}' => $queueCmd,
```

Comment:
@tsingsun, I think, `$this->uniqueId` will be better, because it supports module cascade.